helm/ffc-ce-web/jenkins-aws.yaml

```diff
@@ -0,0 +1,11 @@
+environment: production
+service:
+  port: 443
+  type: NodePort
+container:
+  allowPrivilegeEscalation: false
+  imagePullPolicy: Always
+  paymentUrl: http://ffc-ce-payment-orchestrator.ffc-ce-payment-orchestrator-master
```

Comment:
This will need to change when the backend github project is up and running

Reply:
true, once the back end service is merged the old deployment can be deleted and the url updated. If the back end happens to get merged before I finish these changes I'll update it to save time